build.rs

```diff
@@ -322,6 +341,7 @@ fn ring_build_rs_main() {
 
     let target = Target {
         arch,
+        endian,
```

Comment:
No need to store endian in the target since no big-endian targets are supported.